src/Apps/Order/Routes/Shipping2/ShippingContext.ts

```diff
@@ -1,20 +1,342 @@
-import { FulfillmentValues } from "Apps/Order/Routes/Shipping2/FulfillmentDetails"
-import { createContext, useContext } from "react"
+import {
+  createContext,
+  useCallback,
+  useContext,
+  useMemo,
+  useState,
+} from "react"
+import { FormikProps } from "formik"
+import { ShippingProps, ShippingRouteStep } from "Apps/Order/Routes/Shipping2"
+import {
+  FulfillmentValues,
+  ShipValues,
+} from "Apps/Order/Routes/Shipping2/FulfillmentDetails"
+import {
+  addressWithFallbackValues,
+  getDefaultUserAddress,
+  FulfillmentType,
+  ShippingAddressFormValues,
+} from "Apps/Order/Routes/Shipping2/shippingUtils"
+import { SavedAddressType } from "Apps/Order/Utils/shippingUtils"
+import { extractNodes } from "Utils/extractNodes"
+import { ALL_COUNTRY_CODES, EU_COUNTRY_CODES } from "Components/CountrySelect"
+import { COUNTRIES_IN_EUROPEAN_UNION } from "@artsy/commerce_helpers"
+// TODO: Duplicated list ^
```

Comment:
```
// FIXME: Duplicated list
import { COUNTRIES_IN_EUROPEAN_UNION } from "@artsy/commerce_helpers"
```

Reply:
this is preserved from the legacy route - i just noticed the duplicated collections (via [artsy/address_helper](https://github.com/artsy/address_helper/blob/main/lib/address_helper.rb)) and wanted to note it here. Ultimately I think we should consolidate on this but the comment is just calling out but preserving an existing inconsistency.

Reply:
I understand, but comments typically don't go below the line of code that one is referring to